index.ts

```diff
@@ -99,6 +99,31 @@ async function main() {
             .sort()
             .toString();
           const projectIssueLabelsStringified = getDevpoolIssueLabels(projectIssue, projectUrl).sort().toString();
+
+          // if bounty is unavailable and exists in the list with the state open then close such bounty
+          if (isDevpoolUnavailableLabel && devpoolIssue.state === "open") {
```

Comment:
No need to check the `Unavailable` label

If project (i.e. partner) issue is assigned to somebody and devpool issue from https://github.com/ubiquity/devpool-directory/issues is "open" then close such issue